playground/android/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.taobao.android.weex.plugin.gradle'
 android {
     compileSdkVersion 26
-    buildToolsVersion "25.0.2"
+    buildToolsVersion "28.0.3"
     defaultConfig {
         applicationId "org.weex.demo"
         minSdkVersion 23
```

Comment:
As we have seen the other day, this has to be 18. Otherwise won't work on old/some devices.